src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+import duke.task.*;
+import java.util.List;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+
+    static void checkLineEmpty(String line)throws DukeCheckLineEmptyException{
+        if (line.equals("")){
+            throw new DukeCheckLineEmptyException();
+        }
+    }
+
+    static void checkWord(String line)throws DukeCheckLineException{
```

Comment:
Try to consider similar inputs (e.g. LIST), should Duke not understand what this means?

Reply:
Yes, use exceptions to handle errors.